docs/bundles.md

```diff
@@ -104,9 +105,9 @@ List the instances in the `podinfo` namespace:
 === "output"
 
      ```text
-     NAME    MODULE                                          VERSION LAST APPLIED         
-     podinfo oci://ghcr.io/stefanprodan/modules/podinfo      6.3.5   2023-04-10T16:20:07Z    
-     redis   oci://ghcr.io/stefanprodan/modules/redis        7.0.10  2023-04-10T16:20:00Z
+     NAME    NAMESPACE         MODULE                                          VERSION LAST APPLIED          BUNDLE       
```

Comment:
Can you please add above `timoni list --bundle podinfo -A`

Reply:
Added a dedicated example for name-oriented listing, rather than replacing the namespace-oriented one. Can replace the latter if you'd like.